src/Integration.Vsix/Native/INativeMethods.cs

```diff
@@ -37,20 +43,62 @@ internal class NativeMethods : INativeMethods
         public const uint GetRootWindow = 2;
         public const uint WM_CLOSE = 0x10;
 
+        public const uint SW_SHOWMINIMIZED = 2;
+        public const int SW_RESTORE = 9;
+
         public IntPtr GetAncestor(IntPtr hwnd, uint flags) => NativeMethodDeclarations.GetAncestor(hwnd, flags);
 
         public int SendMessage(IntPtr hWnd, uint Msg, IntPtr wParam, IntPtr lParam) => NativeMethodDeclarations.SendMessage(hWnd, Msg, wParam, lParam);
 
+        public bool GetWindowPlacement(IntPtr hWnd, ref WINDOWPLACEMENT lpwndpl) => NativeMethodDeclarations.GetWindowPlacement(hWnd, ref lpwndpl);
+
+        public bool SetForegroundWindow(IntPtr hWnd) => NativeMethodDeclarations.SetForegroundWindow(hWnd);
+
+        public bool ShowWindow(IntPtr hWnd, int nCmdShow) => NativeMethodDeclarations.ShowWindow(hWnd, nCmdShow);
+
         // Private nested class that contains the native method declarations
         // (the method names are the same as the interface so they can't be
         // declared in the parent class).
         private static class NativeMethodDeclarations
         {
             [DllImport("user32.dll", ExactSpelling = true)]
             public static extern IntPtr GetAncestor(IntPtr hwnd, uint flags);
-            
+
             [DllImport("user32.dll", CharSet = CharSet.Auto)]
             public static extern int SendMessage(IntPtr hWnd, UInt32 Msg, IntPtr wParam, IntPtr lParam);
+
+            [DllImport("user32.dll")]
+            public static extern bool SetForegroundWindow(IntPtr hWnd);
+            [DllImport("user32.dll", SetLastError = true)]
+            public static extern bool ShowWindow(IntPtr hWnd, int nCmdShow);
```

Comment:
If you're making any additional commits then can you please add an empty line here? 😄 (like with the others)